src/environment/environment.dev.ts

```diff
@@ -17,6 +17,6 @@ export const environment : EnvironmentFile = {
     region: 'eu-west-1',
   },
   journal: {
-    journalUrl: 'https://vulv731rce.execute-api.eu-west-1.amazonaws.com/default'
+    journalUrl: 'https://g2u8m8j9o3.execute-api.eu-west-1.amazonaws.com/default'
   }
```

Comment:
Should we all be using this endpoint now?

Reply:
This is slightly up in the air at the moment. Once this PR goes in, you will need to request the data from and endpoint with CORS support for the `If-Modified-Since` header. The one I've switched to here has that support. I am currently working on the Terraform to apply this change to the dev API, but we should probably stay away from there for mainstream use for a little while while platform things stabilise.

Reply:
Keeping this as-is for now to get around token verification until we can work in a dev config credentials mechanism.